modmesh/gui/naca.py

```diff
@@ -94,16 +94,53 @@ def calc_naca4_points(number, npoint, open_trailing_edge=False,
     return ret
 
 
-def draw_naca4(world, number, npoint, fac, off_x, off_y, **kw):
+def calc_sample_points(points):
+    # determine the number of sample points for a cubic bezier curve
+    spacing = 0.01
+    segments = np.hypot((points[:-1, 0] - points[1:, 0]),
+                        (points[:-1, 1] - points[1:, 1])) // spacing
+    nsample = np.where(segments > 2, segments - 1, 2).astype(int)
+    return nsample
+
+
+def linear_approximate(world, points):
+    for it in range(points.shape[0] - 1):
+        e = world.add_edge(points[it, 0], points[it, 1], 0,
+                           points[it + 1, 0], points[it + 1, 1], 0)
+        print(f"{it}: {e}")
```

Comment:
Please help remove debug prints.  The code is being improved and it's time to get rid of them.